validator/rpc/handler_wallet.go

```diff
@@ -4,66 +4,76 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io"
+	"net/http"
 	"path/filepath"
 	"strings"
 
-	"github.com/golang/protobuf/ptypes/empty"
 	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/v4/config/features"
 	"github.com/prysmaticlabs/prysm/v4/io/file"
 	"github.com/prysmaticlabs/prysm/v4/io/prompt"
-	pb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1/validator-client"
+	http2 "github.com/prysmaticlabs/prysm/v4/network/http"
```

Comment:
nit: could you call it something other than http2? http2 is is the package that implements `http/2` protocol (golang.org/x/net/http2)

Reply:
We had a discussion on Slack recently and decided to rename the package to `httputil` to avoid having to always alias it. I'm OK with adding this refactor to the PR

Reply:
ok i can rename this alias

Reply:
not the alias, but the entire package